### PR TITLE
fix(card): themes enabled

### PR DESCRIPTION
**Pull request by greeshmareddyg**
### Related Ticket(s)

https://github.com/carbon-design-system/ibm-dotcom-library/issues/2481

### Changelog

**Changed**

- card - component : enabled themes g90 and g100

### CODE REVIEW

File: packages/react/src/components/Card/Card.js [hunk 1]
```diff
@@ -127,6 +132,11 @@ export const cardPropTypes = {
    */
   heading: PropTypes.string,
 
+  /**
+   * Theme name.
+   */
+  theme: PropTypes.oneOf(['white', 'g10', 'g90', 'g100']),
```
Comment (jeffchew) [inline packages/react/src/components/Card/Card.js:138]: @greeshmareddyg can you also set a defaultProp?

**Comment by jeffchew** [comment]
@greeshmareddyg your snapshots appear to be out of date too. Also check and see @asudoh 's latest on theme switching, we may not need to provide a knob in the story itself anymore.

File: packages/react/src/components/Card/Card.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ import { LinkWithIcon } from '../LinkWithIcon';
 import PropTypes from 'prop-types';
 import React from 'react';
 import settings from 'carbon-components/es/globals/js/settings';
+import TableOfContents from '../TableOfContents/TableOfContents';
```
Comment (jeffchew) [inline packages/react/src/components/Card/Card.js:22]: @greeshmareddyg this might have been a mistake to add `TableofContents` here?

Comment (greeshmareddyg) [reply packages/react/src/components/Card/Card.js]: yeah, updated Jeff.

**Comment by asudoh** [comment]
> Also check and see @asudoh 's latest on theme switching, we may not need to provide a knob in the story itself anymore.

Good point, wondering if we still need a new React prop.

**Comment by jeffchew** [comment]
@greeshmareddyg can you look at the merge conflicts?

**Comment by greeshmareddyg** [comment]
@jeffchew @asudoh I think we don't need to merge this, because the recent theme changes taking care of the theme for the component.

**Comment by jeffchew** [comment]
Thanks @greeshmareddyg ! Will close this then.

**Pull request closed without merging**